test/conftest.py

```diff
@@ -25,16 +43,32 @@ def pytest_addoption(parser):
 @pytest.fixture(scope='session')
 def config(request):
     return {
-        'TEST_DB_URI': request.config.getoption('TEST_DB_URI')
+        SQLITE_TEST_DB_URI: request.config.getoption(SQLITE_TEST_DB_URI),
+        MYSQL_TEST_DB_URI: request.config.getoption(MYSQL_TEST_DB_URI),
     }
 
 
-@pytest.fixture(scope='session')
-def db_uri(config):
-    return config['TEST_DB_URI']
+def test_db_keys():
+    """Decide what DB backends to use to run the tests."""
+    test_db_uris = []
+    test_db_uris.append(SQLITE_TEST_DB_URI)
+
+    try:
+        import mysql  # noqa: F401
+    except ImportError:
+        pass
+    else:
+        test_db_uris.append(MYSQL_TEST_DB_URI)
```

Comment:
I tried to use `skipif` as you suggested earlier, but since all the tests are exactly the same on both DB backends I did not see how I could easily do it with pytest.

I want to prevent tests from running twice but I don't want to skip them completely, how do you think I could do it with `skipif` since it's applied directly to a test function or class?

As I said earlier, I will merge this and will change it to `skipif` as part of another PR once I see how.